### PR TITLE
docs(tools): add repl section

### PR DESCRIPTION
**Pull request by caspervonb**
This adds a section on the repl with the keybindings that we support out of the box

### CODE REVIEW

File: docs/tools/repl.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+up program state in the global context.
+
+## Keybindings
+
+| Keystroke             | Action                                                                                           |
+| --------------------- | ------------------------------------------------------------------------------------------------ |
+| End                   | Move cursor to end of line                                                                       |
+| Home                  | Move cursor to the beginning of line                                                             |
+| Left                  | Move cursor one character left                                                                   |
+| Right                 | Move cursor one character right                                                                  |
```
Comment (caspervonb) [inline docs/tools/repl.md:13]: Duplicated

```suggestion
```

**Review by crypt096 — APPROVED** [review]

**Comment by David-Else** [comment]
Nice one!

- You might want to explain or rephrase 'yank' for people who don't use Vim?
- `Interrupt/Cancel edition`: I am not sure what an `edition` is, just me being ignorant?
- Maybe explain what a `meta` key is? https://www.computerhope.com/jargon/m/meta-key.htm, I had not heard the term before

Maybe it is just me not knowing these terms, but new users might also get confused.

**Review by bartlomieju — APPROVED** [review]
LGTM

**Comment by caspervonb** [comment]
> Maybe it is just me not knowing these terms, but new users might also get confused.

When it comes to yank/kill etc, terminology tends to derived from emacs/vi terminology but open for revision of course 👍

**Pull request merged**